packages/react/src/components/Loader/Loader.tsx

```diff
@@ -51,27 +61,61 @@ class Loader extends UIComponent<ReactProps<LoaderProps>> {
       content: false,
       color: true,
     }),
+    delay: PropTypes.number,
     indicator: customPropTypes.itemShorthand,
+    inline: PropTypes.bool,
     label: customPropTypes.itemShorthand,
     labelPosition: PropTypes.oneOf(['above', 'below', 'start', 'end']),
     size: customPropTypes.size,
   }
 
   static defaultProps = {
     accessibility: loaderBehavior,
+    delay: 0,
```

Comment:
Should be provide a more reasonable default? Maybe 100ms?

Reply:
By default it should `0` to keep existing behavior.